plugins/entando-plugin-jacms/src/main/webapp/WEB-INF/plugins/jacms/apsadmin/jsp/resource/cropEditorModal.jsp

```diff
@@ -0,0 +1,252 @@
+                                                            data-method="remove"
+                                                            title="Remove">
+                                                        <span class="docs-tooltip" data-toggle="tooltip" title=""
+                                                              data-original-title="cropper.clear()">
+                                                            <span class="fa fa-remove"></span>
+                                                        </span>
+                                                    </button>
+                                                </div>
+                                            </div>
+                                        </div>
+                                        <div class="col-md-4 aspect-ratio-buttons-container ">
+                                            <div class="aspect-ratio-buttons">
+                                                <div class="btn-group d-flex flex-nowrap" data-toggle="buttons">
+                                                    <label class="btn btn-primary active"
+                                                           data-method="setAspectRatio" data-option="NaN">
+                                                        <input type="radio" class="sr-only" id="aspectRatio5"
+                                                               name="aspectRatio" value="NaN">
+                                                        <span class="docs-tooltip" data-toggle="tooltip" title=""
+                                                              data-original-title="aspectRatio: NaN">Free</span>
+                                                    </label>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <!-- /tab pane blue print -->
+                        </div>
+                    </div>
+                    <div class="col-md-4">
+                        <!-- Nav tabs -->
+                        <ul class="nav nav-tabs tabs-left image-navigation">
+                            <li>List of images</li>
+
+                        </ul>
+                        <!-- image navigation item blueprint -->
+                        <li class="active image-navigation-item hidden" id="image-navigation-item-blueprint">
+                            <a href="#first" data-toggle="tab">Blueprint</a></li>
+                        <!-- /image navigation item blueprint -->
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+        <div id="aspect-ratio-values">
+            <wp:info key="systemParam" paramName="aspect_ratio"/>
+        </div>
+
+
+    </div>
+</div>
```

Comment:
Hey @otarza  this modal should be trannslated in italian as well, could you please add dynamic labels instead of static text? Let me know if you need help for the italian translations.

Reply:
Hi @alepintus, 

Sure, let me check and if I need help will let you know. 

Thanks.